src/ao_store.erl

```diff
@@ -13,11 +13,33 @@
 %%% {error, no_viable_store}.
 
 behavior_info(callbacks) ->
-    [
-        {start, 1}, {stop, 1}, {reset, 1}, {make_group, 2}, {make_link, 3},
-        {type, 2}, {read, 2}, {write, 3},
-        {list, 2}, {path, 2}, {add_path, 3}
-    ].
+	[
+		% starts. 0 usages
+		{start, 1},
+		% stops. 0 usages
+		{stop, 1},
+		% removes the data. 1 usage
+		{reset, 1},
+
+		% just does ensure_dir
+		{make_group, 2},
+		% creates a simlink
+		{make_link, 3},
+		% can be composite(I guess nested keys) //simple//notfound
```

Comment:
Please can we use capital letters at the start of comments in the code? The rest of the codebase is written like this

Reply:
I have reverted this file to the original state, as it was not supposed to be a part of this commit. However, will follow the comments convention.